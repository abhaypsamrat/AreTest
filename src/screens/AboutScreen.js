import {View, Text, Button} from 'react-native';
import React from 'react';

const AboutScreen = () => {
  return (
    <View>
      <Text>About Screen</Text>
      <Button title="Button" onPress={() => {}}/>
      <View>
        <Text>Hello</Text>
      </View>
    </View>
  );
};

export default AboutScreen;
