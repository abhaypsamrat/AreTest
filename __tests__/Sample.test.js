import React from 'react';
import {render} from '@testing-library/react-native';
import renderer from 'react-test-renderer';
import AboutScreen from '../src/screens/AboutScreen';

describe('This is for test suite', () => {
  test('this is for the test', () => {
    const {getByText} = render(<AboutScreen />);
    expect(getByText('About Screen')).toBeTruthy();
  });

  test('this test case match snapshots', () => {
    const tree = renderer.create(<AboutScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
