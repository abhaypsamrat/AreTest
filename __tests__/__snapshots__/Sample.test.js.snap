// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This is for test suite this test case match snapshots 1`] = `null`;
